src/main/resources/templates/courselist.twig

```diff
@@ -4,8 +4,8 @@
     <div class="list-group courselist">
         <p class="list-group-item active">Course List</p>
         {% for course in courseList %}
-            <a href="/{{ course.instructor.username }}/{{ course.name }}{% if course.section %}/{{ course.section }}{% endif %}"
-               class="list-group-item courselist-item">{{ course.getName() }}</a>
+            <a href="/{{ course.instructor.username }}/{{ course.name }}{% if course.getSection.equals("") %} {% else %}/{{ course.section }}{% endif %}"
+               class="list-group-item courselist-item">{{ course.getName() }}{% if course.getSection.equals("") %} {% else %} - {{ course.getSection() }}{% endif %}</a>
```

Comment:
could you not use negation instead?

`{% if not course.getSection.equals("") %}{{ course.getSection() }}{% endif %}`

Reply:
We had this originally, but it did not work. We also tried 
`{% if !course.getSection.equals("") %}{{ course.getSection() }}{% endif %}` 
which did not work either

Reply:
Could it be be because you're using `getSection` and not `getSection()`?

Why not do this for the first one? `{{ course.getSection() }}`? If the section is empty, it will just inject the empty string, which is fine.

For the second one, we can just do this (the `-` isn't super necessary): `{{ course.getSection() }}`